Xwt/Xwt/FileSelector.cs

```diff
@@ -169,10 +169,11 @@ public DefaultFileSelectorBackend ()
 			var box = new HBox ();
 			box.Accessible.IsAccessible = true;
 			box.Accessible.Role = Accessibility.Role.Group;
-			box.Accessible.Title = Application.TranslationCatalog.GetString("File Selector");
+			box.Accessible.Description = Application.TranslationCatalog.GetString("Enter File Selector");
 
 			entry = new TextEntry ();
-			entry.Accessible.Label = Application.TranslationCatalog.GetString ("Path");
+			entry.Accessible.Label = Application.TranslationCatalog.GetString ("Enter File Path");
```

Comment:
Same here, since it's a text entry, it's obvious that it's made for entering text, "File Path" (label>purpose) in this case.